libs/wingc/src/jsify.rs

```diff
@@ -1655,8 +1650,14 @@ impl<'a> FieldReferenceVisitor<'a> {
 				// To obtain information about the variable we're referencing (like its type and
 				// whether it's reassignable), we look it up in the function's symbol environment.
 				let var = env
-					.lookup(&x, Some(self.statement_index), 0)
-					.expect("covered by type checking")
+					.lookup(&x, Some(self.statement_index))
+					.expect(
+						format!(
+							"unable to find symbol '{x}' in environment that contains: {}",
+							env.symbol_map.keys().join(", ")
+						)
+						.as_str(),
```

Comment:
This will also work
```suggestion
						&format!(
							"unable to find symbol '{x}' in environment that contains: {}",
							env.symbol_map.keys().join(", ")
						),
```
But anyway, this is one heck of a verbose `expect`... maybe too verbose.